src/sql/workbench/parts/profiler/browser/profilerEditor.ts

```diff
@@ -336,13 +336,23 @@ export class ProfilerEditor extends BaseEditor {
 		tabbedPanelContainer.className = 'profiler-tabbedPane';
 		this._tabbedPanel = new TabbedPanel(tabbedPanelContainer);
 		attachTabbedPanelStyler(this._tabbedPanel, this.themeService);
+
+		const expandPanel = () => {
+			if (this._collapsedPanelAction.collapsed) {
+				this._collapsedPanelAction.run(this.input);
+			}
+		};
+
 		this._tabbedPanel.pushTab({
 			identifier: 'editor',
 			title: nls.localize('text', "Text"),
 			view: {
 				layout: dim => this._editor.layout(dim),
 				render: parent => parent.appendChild(editorContainer),
 				focus: () => this._editor.focus()
+			},
```

Comment:
not 100% sure, but I think the syntax `tabSelectedHandler: expandPanel` works here without needing another wrapper function.